incubator/binding-asyncapi.spec/src/main/scripts/io/aklivity/zilla/specs/binding/asyncapi/config/client.tls.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: test
+bindings:
+  asyncapi0:
+    type: asyncapi
+    kind: client
+    options:
+      specs:
+        - mqtt/asyncapi.yaml
+      tcp:
+        host: localhost
+        port:
+          - 7183
+      tls:
+        trust:
+          - serverca
```

Comment:
`trust` requires a `vault`, so please add a `test` vault here and reference it from the binding `vault` property too